src/pages/slash-graphql/administering-your-backend.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Administering Your Slash GraphQL Backend'
+---
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+*These are draft docs for Slash GraphQL, which is currently in beta*
+
+Here is a guide to programatically administering your Slash GraphQL backend. Wherever possible, we have maintained compatibility with the corresponding Dgraph API, with the additional step of requiring authentication via the 'X-Auth-Token' header.
+
+## Authentication and Endpoint
+
+All the APIs documented here require an API token for access. A new API token can be generated from Slash GraphQL by selecting the ["Settings" button](https://slash.dgraph.io/_/settings) from the sidebar, then clicking the Add API Key button. Do remember to keep your API key safely, as we will not show you the same API key again.
```

Comment:
Do remember to keep your API key safely, as we will not show you the same API key again.
Can be:
Keep your API key safe, it will not be accessible once you leave the page.